doc/ref_impl/cntt-ri2/chapters/chapter01.rst

```diff
@@ -0,0 +1,139 @@
+Overview
+========
+
+Introduction
+------------
+
+About the Kubernetes Reference Implementation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This document includes the requirements and deployment details of the Kubernetes-based Reference Implementation (RI2). This RI will conform to the architecture specification as defined in the `Kubernetes Based Reference Architecture (RA2) <../../../ref_arch/kubernetes/README.md>`__, including alignment with the principles and requirements of the `Reference Model <../../../ref_model/README.md>`__.
```

Comment:
Can we define these references at the end like the figures? Easy to change if a reference link changes or there is an error, etc. Thanks.

Reply:
yes. rst typically makes use of named references, i.e., there is a global name defined at the target and that is being referred to from everywhere. However, we should introduce this over time as it requires the target anchors to exist.